### PR TITLE
docs: add 4.14.0 changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 4.14.0

`2021-03-14`

- 🐞 Fix Tabs `centered` prop is not actually center. [#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
- 🐞 Fix AutoComplete custom input `className` missing. [#29725](https://github.com/ant-design/ant-design/pull/29725)
- 💄 Fixed console warning when setting `margin` style on Row. [#29688](https://github.com/ant-design/ant-design/pull/29688)
- Upload
  - 🐞 Fix Upload `onChange` params `file` can not `cloneDeep` by lodash. [#29718](https://github.com/ant-design/ant-design/pull/29718)
  - 🐞 Fix Upload crash when `fileList` is `null`. [#29702](https://github.com/ant-design/ant-design/pull/29702)
  - 💄 Upload motion add 2s deadline to avoid developer manually remove motion makes hanging. [#29686](https://github.com/ant-design/ant-design/pull/29686)
- 🐞 Fix the bug that Modal footer buttons not spaced properly when using href in button. [#29681](https://github.com/ant-design/ant-design/pull/29681) [@n0ruSh](https://github.com/n0ruSh)
- 💄 Fixed the error style where the `disabled` Input has affix elements. [#29670](https://github.com/ant-design/ant-design/pull/29670)
- 💄 Optimize the cursor style of Form.Item tooltip info. [#29650](https://github.com/ant-design/ant-design/pull/29650)
- - 🌐 fix typo in cs-CZ locale. [#29675](https://github.com/ant-design/ant-design/pull/29675) [@jvaclavik](https://github.com/jvaclavik)

--------

## 4.14.0

`2021-03-14`

- 🐞 修复 Tabs 设置 `centered` 后居中位置有便宜。[#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
- 🐞 修复 AutoComplete 自定义 input 上 `className` 属性丢失的问题。[#29725](https://github.com/ant-design/ant-design/pull/29725)
- 💄 修复 Row 设置 `margin` 样式时控制台警告的问题。[#29688](https://github.com/ant-design/ant-design/pull/29688)
- Upload
  - 🐞 修复 Upload `onChange` 参数 `file` 不能被 lodash `cloneDeep` 的问题。[#29718](https://github.com/ant-design/ant-design/pull/29718)
  - 🐞 修复 Upload 当 `fileList` 为 `null` 时崩溃的问题。[#29702](https://github.com/ant-design/ant-design/pull/29702)
  - 💄 Upload 添加 2 秒时限以防止开发者手工移除动画样式时导致动画卡住的问题。[#29686](https://github.com/ant-design/ant-design/pull/29686)
- 🐞 修复 Modal 页脚里使用 href 按钮导致的间距丢失问题。[#29681](https://github.com/ant-design/ant-design/pull/29681) [@n0ruSh](https://github.com/n0ruSh)
- 💄 修复 Input 组件配置附件元素时禁用样式异常的问题。[#29670](https://github.com/ant-design/ant-design/pull/29670)
- 💄 优化 Form.Item 提示信息的鼠标显示样式。[#29650](https://github.com/ant-design/ant-design/pull/29650)
- 🌐 修复 cs-CZ 语言环境中的错字。 [#29675](https://github.com/ant-design/ant-design/pull/29675) [@jvaclavik](https://github.com/jvaclavik)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
+
+- 🐞 Fix Tabs `centered` prop is not actually center. [#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
+- 🐞 Fix AutoComplete custom input `className` missing. [#29725](https://github.com/ant-design/ant-design/pull/29725)
+- 💄 Fixed console warning when setting `margin` style on Row. [#29688](https://github.com/ant-design/ant-design/pull/29688)
+- Upload
+  - 🐞 Fix Upload `onChange` params `file` can not `cloneDeep` by lodash. [#29718](https://github.com/ant-design/ant-design/pull/29718)
+  - 🐞 Fix Upload crash when `fileList` is `null`. [#29702](https://github.com/ant-design/ant-design/pull/29702)
+  - 💄 Upload motion add 2s deadline to avoid developer manually remove motion makes hanging. [#29686](https://github.com/ant-design/ant-design/pull/29686)
+- 🐞 Fix the bug that Modal footer buttons not spaced properly when using href in button. [#29681](https://github.com/ant-design/ant-design/pull/29681) [@n0ruSh](https://github.com/n0ruSh)
+- 💄 Fixed the error style where the `disabled` Input has affix elements. [#29670](https://github.com/ant-design/ant-design/pull/29670)
```
Comment (xrkffgg) [inline CHANGELOG.en-US.md:30]:
```suggestion
- 💄 Fix the error style where the `disabled` Input has affix elements. [#29670](https://github.com/ant-design/ant-design/pull/29670)
```

**Comment by xrkffgg** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/29615
- [ ] https://github.com/ant-design/ant-design/pull/29634
- [ ] https://github.com/ant-design/ant-design/pull/29737

**Comment by chenshuai2144** [comment]
我重新合并一下 featrue。。。

**Comment by afc163** [comment]
为啥月中发 minor？

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,27 @@ timeline: true
 
 ---
 
+## 4.14.0
+
+`2021-03-14`
+
+- Tabs
+  - 🆕 Tabs 新增 `moreIcon` 参数。[#29744](https://github.com/ant-design/ant-design/pull/29744) [@tianyuan233](https://github.com/tianyuan233)
+  - 🐞 修复 Tabs 设置 `centered` 后居中位置有便宜。[#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:24]: 偏移

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
+- 💄 修复 Row 设置 `margin` 样式时控制台警告的问题。[#29688](https://github.com/ant-design/ant-design/pull/29688)
+- Upload
+  - 🆕 重构 Upload 使用浏览器的原生行为。[#29737](https://github.com/ant-design/ant-design/pull/29737)
+  - 🐞 修复 Upload `onChange` 参数 `file` 不能被 lodash `cloneDeep` 的问题。[#29718](https://github.com/ant-design/ant-design/pull/29718)
+  - 🐞 修复 Upload 当 `fileList` 为 `null` 时崩溃的问题。[#29702](https://github.com/ant-design/ant-design/pull/29702)
+  - 💄 Upload 添加 2 秒时限以防止开发者手工移除动画样式时导致动画卡住的问题。[#29686](https://github.com/ant-design/ant-design/pull/29686)
+- 🐞 修复 Modal 页脚里使用 href 按钮导致的间距丢失问题。[#29681](https://github.com/ant-design/ant-design/pull/29681) [@n0ruSh](https://github.com/n0ruSh)
+- 💄 修复 Input 组件配置附件元素时禁用样式异常的问题。[#29670](https://github.com/ant-design/ant-design/pull/29670)
+- 💄 优化 Form.Item 提示信息的鼠标显示样式。[#29650](https://github.com/ant-design/ant-design/pull/29650)
+- 🌐 修复 cs-CZ 语言环境中的错字。 [#29675](https://github.com/ant-design/ant-design/pull/29675) [@jvaclavik](https://github.com/jvaclavik)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:36]: 下划线。

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
+- 🌐 添加 fr_CA 语言。[#29748](https://github.com/ant-design/ant-design/pull/29748) [@yoyo83](https://github.com/yoyo837)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:37]: 国旗。

**Comment by chenshuai2144** [comment]
这个 Upload 的修改影响很大，让不少系统挂了，希望回滚掉

**Comment by afc163** [comment]
> 这个 Upload 的修改影响很大，让不少系统挂了，希望回滚掉

这个和版本号的关系是？

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
+  - 🐞 Fix Upload `onChange` params `file` can not `cloneDeep` by lodash. [#29718](https://github.com/ant-design/ant-design/pull/29718)
+  - 🐞 Fix Upload crash when `fileList` is `null`. [#29702](https://github.com/ant-design/ant-design/pull/29702)
+  - 💄 Upload motion add 2s deadline to avoid developer manually remove motion makes hanging. [#29686](https://github.com/ant-design/ant-design/pull/29686)
+- 🐞 Fix the bug that Modal footer buttons not spaced properly when using href in button. [#29681](https://github.com/ant-design/ant-design/pull/29681) [@n0ruSh](https://github.com/n0ruSh)
+- 💄 Fix the error style where the `disabled` Input has affix elements. [#29670](https://github.com/ant-design/ant-design/pull/29670)
+- 💄 Optimize the cursor style of Form.Item tooltip info. [#29650](https://github.com/ant-design/ant-design/pull/29650)
+- 🌐 Fix typo in cs-CZ locale. [#29675](https://github.com/ant-design/ant-design/pull/29675) [@jvaclavik](https://github.com/jvaclavik)
+- 🌐 Add fr_CA locale. [#29748](https://github.com/ant-design/ant-design/pull/29748) [@yoyo837](https://github.com/yoyo837)
```
Comment (afc163) [inline CHANGELOG.en-US.md:37]:
```suggestion
- 🌐 Add fr_CA locale. [#29748](https://github.com/ant-design/ant-design/pull/29748) [@liufenghua808](https://github.com/liufenghua808)
```

**Comment by chenshuai2144** [comment]
@afc163 改动特别大，可能会把改好又弄挂，所以豆酱老师让我发个 minor

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
+`2021-03-14`
+
+- Tabs
+  - 🆕 Tabs 新增 `moreIcon` 参数。[#29744](https://github.com/ant-design/ant-design/pull/29744) [@tianyuan233](https://github.com/tianyuan233)
+  - 🐞 修复 Tabs 设置 `centered` 后居中位置偏移。[#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
+- 🐞 form 表单现在可以自动响应 reset 事件。[#29752](https://github.com/ant-design/ant-design/pull/29752) [@jueinin](https://github.com/jueinin)
+- 🐞 修复 AutoComplete 自定义 input 上 `className` 属性丢失的问题。[#29725](https://github.com/ant-design/ant-design/pull/29725)
+- 💄 修复 Row 设置 `margin` 样式时控制台警告的问题。[#29688](https://github.com/ant-design/ant-design/pull/29688)
+- Upload
+  - 🆕 重构 Upload 使用浏览器的原生行为。[#29737](https://github.com/ant-design/ant-design/pull/29737)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:29]: 这句话不太准确，和豆酱沟通一下。

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/29760#issuecomment-798868597

这里没补齐呀，我看 #29615 就没有。

**Comment by chenshuai2144** [comment]
29752  和 29615 看混了，我加一下

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
 ---
 
+## 4.14.0
+
+`2021-03-14`
+
+- Tabs
+  - 🐞 Fix Tabs `centered` prop is not actually center. [#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
+  - 🆕 Tabs support `moreIcon`. [#29744](https://github.com/ant-design/ant-design/pull/29744) [@tianyuan233](https://github.com/tianyuan233)
+- 🐞 button with htmlType='reset' will reset all form fileds. [#29752](https://github.com/ant-design/ant-design/pull/29752) [@jueinin](https://github.com/jueinin)
```
Comment (afc163) [inline CHANGELOG.en-US.md:25]:
```suggestion
- 🐞 Buttons with `htmlType="reset"` will reset all form fileds now. [#29752](https://github.com/ant-design/ant-design/pull/29752) [@jueinin](https://github.com/jueinin)
```

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
 ---
 
+## 4.14.0
+
+`2021-03-14`
+
+- Tabs
+  - 🆕 Tabs 新增 `moreIcon` 参数。[#29744](https://github.com/ant-design/ant-design/pull/29744) [@tianyuan233](https://github.com/tianyuan233)
+  - 🐞 修复 Tabs 设置 `centered` 后居中位置偏移。[#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
+- 🐞 form 表单现在可以自动响应 reset 事件。[#29752](https://github.com/ant-design/ant-design/pull/29752) [@jueinin](https://github.com/jueinin)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:25]:
```suggestion
- 🐞 Form 表单现在可以自动响应 reset 事件。[#29752](https://github.com/ant-design/ant-design/pull/29752) [@jueinin](https://github.com/jueinin)
```

**Comment by afc163** [comment]
#29634 也没有。

File: CHANGELOG.en-US.md [hunk 9]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+
+- 🆕 Add parent class for different Notification types. [#29634](https://github.com/ant-design/ant-design/pull/29634) [@n0ruSh](https://github.com/n0ruSh)
+- 🆕 Typography editable supports `onCancel` and `onEnd`. [#29615](https://github.com/ant-design/ant-design/pull/29615) [@jueinin](https://github.com/jueinin)
+- Tabs
+  - 🐞 Fix Tabs `centered` prop is not actually center. [#29495](https://github.com/ant-design/ant-design/pull/29495) [@jinchaofs](https://github.com/jinchaofs)
+  - 🆕 Tabs support `moreIcon`. [#29744](https://github.com/ant-design/ant-design/pull/29744) [@tianyuan233](https://github.com/tianyuan233)
+- 🐞 Button with `htmlType='reset'` will reset all form fileds. [#29752](https://github.com/ant-design/ant-design/pull/29752) [@jueinin](https://github.com/jueinin)
+- 🐞 Fix AutoComplete custom input `className` missing. [#29725](https://github.com/ant-design/ant-design/pull/29725)
+- 💄 Fix console warning when setting `margin` style on Row. [#29688](https://github.com/ant-design/ant-design/pull/29688)
+- Upload
```
Comment (afc163) [inline CHANGELOG.en-US.md:30]: Upload 改动最大，放前面。

Comment (chenshuai2144) [reply CHANGELOG.en-US.md:30]: ok

File: CHANGELOG.zh-CN.md (hunk 6 above)
Comment (chenshuai2144) [reply CHANGELOG.zh-CN.md]: 豆酱老师准备了一个巨长的文案

**Comment by yoyo837** [comment]
> > 这个 Upload 的修改影响很大，让不少系统挂了，希望回滚掉
> 
> 这个和版本号的关系是？

应该是Upload应该发minor而不是patch的意思

**Pull request merged**